src/include/stat.h

```diff
@@ -159,10 +159,10 @@ __wt_stats_clear(void *stats_arg, int slot)
         if (WT_STAT_ENABLED(session))                                 \
             (void)__wt_atomic_subi64(&(stat)->fld, (int64_t)(value)); \
     } while (0)
-#define WT_STAT_INCRV_BASE(session, stat, fld, value) \
-    do {                                              \
-        if (WT_STAT_ENABLED(session))                 \
-            (stat)->fld += (int64_t)(value);          \
+#define WT_STAT_INCRV_BASE(session, stat, fld, value)           \
```

Comment:
We've suppressed all stats code, but this is popping up in our mutex code so I've made it atomic.
I'll leave some perf tests to run overnight and confirm this is safe.

Reply:
So we'd be modifying just one of the 4ish stat macros?

Reply:
Could we do ifdef tsan use inline funcs for statistics?

Reply:
There was a visible perf hit in the most recent tests so I removed this change and tried to suppress in a different manner, however I haven't been able to reproduce it. 
Since perf is back to normal and I can't get the tests to fail when WT_STAT_INCRV_BASE is non-atomic. I'll leave it as is and come back to this when I can find a reliable repro.